src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,52 @@
+.calendar {
+  display: flex;
+  flex-shrink: 0;
+  flex-wrap: wrap;
+  gap: $gap-between-columns;
+  padding: $calendar-padding;
+  width: ($column-size * $columns-in-row) +
+    ($gap-between-columns * $gaps-in-row) +
+    $calendar-padding;
+
+  @for $days from 28 through 31 {
+    &--month-length--#{$days} {
+      :nth-child(n + #{$days+1}) {
+        visibility: hidden;
+      }
+    }
+  }
+
+  @for $i from 1 through 31 {
+    :nth-child(#{$i})::before {
+      content: "#{$i}";
+      display: block;
+      font-family: Arial, Helvetica, sans-serif;
+      font-size: 30px;
+      line-height: $column-size;
+      text-align: center;
```

Comment:
Leave only content here. You can add all other properties using `calendar__day` selector, no need to use loop for that
![image](https://github.com/mate-academy/layout_calendar/assets/96409563/6c34dab2-0585-4323-a5b1-22315c614ef7)